modules/casper.js

```diff
@@ -214,6 +216,55 @@ var Casper = function Casper(options) {
         */
     });
 
+    // management ResourceTimeout with slimerJS
+    if (phantom.casperEngine === "slimerjs") {
+        this.checkTimeout = (function (data) {
+            var self = data;
+            return function(){
+                if ((!!this.page.settings.resourceTimeout) && (!isNaN(this.page.settings.resourceTimeout))) {
+                    var d = new Date().getTime();
+                    for (var i = self.resourcesTime.length-1 ; i >= 0 ; i--) {
+                        if (d > self.resourcesTime[i].time + self.page.settings.resourceTimeout) {
+                            var request = self.resourcesTime[i];
+                            self.resourcesTime.splice(i, 1);
+                            request.errorCode = 408;
+                            request.errorString = "Network timeout on resource.";
```

Comment:
I just copy paste the phantomjs error message : "Network timeout on resource." Are you sure to want a so long error text ?